prov/util/src/util_cq.c

```diff
@@ -90,22 +90,6 @@ int fi_check_cq_attr(const struct fi_provider *prov,
 	return 0;
 }
 
-static void util_cq_progress(struct util_cq *cq)
-{
-	struct util_ep *ep;
-	struct fid_list_entry *fid_entry;
-	struct dlist_entry *item;
-
-	fastlock_acquire(&cq->list_lock);
-	dlist_foreach(&cq->list, item) {
-		fid_entry = container_of(item, struct fid_list_entry, entry);
-		ep = container_of(fid_entry->fid, struct util_ep, ep_fid.fid);
-		ep->progress(ep);
-
-	}
-	fastlock_release(&cq->list_lock);
-}
-
```

Comment:
Can the above function be left in the util code and made available to use for other providers? Udp provider can pass this as function pointer. The rxm provider could possibly use this progress function instead of implementing its own.